### PR TITLE
support python3

### PR DESCRIPTION
**Pull request by kse201**
add tweaks for python 3.X.
please review it.

travis build : [Build Status](https://travis-ci.org/kse201/nose-blacklist)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,11 @@
 [tox]
-envlist = py27, flake8
+envlist = py27, py37, flake8
```
Comment (pglass) [inline tox.ini]: Please add py36 here also

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 language: python
-
 env:
     - TOXENV=py27
+    - TOXENV=py36
```
Comment (pglass) [inline .travis.yml]: Please add `TOXENV=py37` here, to have travis test with 3.7 as well

**Review by pglass — COMMENTED** [review]
I haven't tested yet, but it looks good. Thank you! Just two small comments

File: tox.ini (hunk 1 above)
Comment (kse201) [reply tox.ini]: ooops! I did typo

**Comment by kse201** [comment]
thanks review!
I fixed reviewed points and add more tweaks because travis default python version is still 3.6.

**Review by pglass — APPROVED** [review]

**Comment by pglass** [comment]
Merged and released! https://pypi.org/project/nose-blacklist/0.0.2/

Let me know if you find any issues. Thank you!

**Pull request merged**